aws-aurora-mysql.yml

```diff
@@ -31,7 +31,7 @@ provision:
     default: csbdb
     constraints:
       maxLength: 64
-      pattern: ^[a-z][a-z0-9-]+$
+      pattern: ^[a-z][a-z0-9]+$
```

Comment:
Underscore is also an allowed char for the db name in both services .

When putting an invalid name the error message is: 
`Invalid character, use letters, numbers, and _`